docs/source/tracking.rst

```diff
@@ -497,33 +497,6 @@ By default ``--backend-store-uri`` is set to the local ``./mlruns`` directory (t
 running ``mlflow run`` locally), but when running a server, make sure that this points to a
 persistent (that is, non-ephemeral) file system location.
 
-Artifact Stores
```

Comment:
Thanks for fixing this duplicated heading! Could you move the new, consolidated `Artifact Stores` section up here? It should go above `File store performance`, which is not as useful to know about, and `Deletion behavior`, which mentiones the artifact store.

Reply:
updated